far2l/src/Mounts.cpp

```diff
@@ -108,10 +108,10 @@ namespace Mounts
 		FileSizeToStr(val, mp.avail, -1, COLUMN_ECONOMIC | COLUMN_FLOATSIZE | COLUMN_SHOWBYTESINDEX);
 		ReplaceStrings(str, L"$A", val);
 
-		FileSizeToStr(val, mp.freee, -1, COLUMN_ECONOMIC | COLUMN_FLOATSIZE | COLUMN_SHOWBYTESINDEX);
+		FileSizeToStr(val, mp.free, -1, COLUMN_ECONOMIC | COLUMN_FLOATSIZE | COLUMN_SHOWBYTESINDEX);
```

Comment:
These 3 eee were there intentionally - to be compatible with systems where free() is a macro for example. Suggest to still use some different name, like with trailing underscore if eee is really bad :)